### ISSUE TITLE
Setting `source` and `destination` to the same path isn't allowed!

### ISSUE THREAD

**Issue by ltartari** [jgarber623/svgeez#16]
I run into this message while trying to build using the latest version (0.2.6):

Output:
```
$ svgeez build --source app/assets/images/icons/ --destination app/assets/images/icons.svg
Setting `source` and `destination` to the same path isn't allowed!
```

As you will see below, this was my expected output (from 0.2.3, the last working version for me):
```
$ svgeez build --source app/assets/images/icons/ --destination app/assets/images/icons.svg
Generating sprite at `/Users/leotartari/src/[REDACTED]/app/assets/images/icons.svg` from 106 SVGs...
Successfully generated sprite at `/Users/leotartari/src/[REDACTED]/app/assets/images/icons.svg`.
```

Please let me know if I have missed something or if you need anything else to reproduce it.

**Comment by bwilkins** [comment]
Have submitted https://github.com/jgarber623/svgeez/pull/17 to fix this issue.

**Comment by jgarber623** [comment]
Ooph, sorry about that, @ltartari!

And thanks for the fix, @bwilkins! I've taken the fix only (a492287) and published a new version of the gem. Will comment on #17 about getting those test additions in.
